shared/models/jobsPartnersComputed.model.ts

```diff
@@ -14,6 +14,7 @@ export enum COMPUTED_ERROR_SOURCE {
   API_ADRESSE = "api_adresse",
   API_ROMEO = "api_romeo",
   VALIDATION = "validation",
+  RANKING = "RANKING",
```

Comment:
```suggestion
  RANKING = "ranking",
```
pour conserver la typo avec les autres champs